domains/certificates/Query.API/API.IntegrationTests/Factories/QueryApiWebApplicationFactory.cs

```diff
@@ -210,7 +210,8 @@ private string GenerateB2CDummyToken(
         {
             { UserClaimName.Scope, scope },
             { JwtRegisteredClaimNames.Name, name },
-            { ClaimType.OrgIds, orgId },
+            { ClaimType.OrgId, orgId },
+            { ClaimType.OrgIds, "" },
```

Comment:
Skal det være en empty string, eller en string med en empty list?

Reply:
in the token it should be an empty string.